projects/plugins/jetpack/extensions/blocks/premium-content/_inc/subscription-service/include.php

```diff
@@ -70,8 +68,8 @@ function default_service( $service ) {
 		return new WPCOM_Online_Subscription_Service();
```

Comment:
just to understand, what was the difference between `WPCOM_Online_Subscription_Service()`, `WPCOM_Token_Subscription_Service`?

Reply:
`WPCOM_Token_Subscription_Service` is a class which `WPCOM_Online_Subscription_Service` inherited from. `WPCOM_Token_Subscription_Service` used to inherit from `Token_Subscription_Service`, just as `Jetpack_Token_Subscription_Service` does currently.

`WPCOM_Token_Subscription_Service` and `Jetpack_Token_Subscription_Service` were so similar that can be refactored into one class, which is what this PR does. That then makes `WPCOM_Online_Subscription_Service` inherit from `Jetpack_Token_Subscription_Service` instead of `Token_Subscription_Service`.

Reply:
thanks for the explanation Chris!